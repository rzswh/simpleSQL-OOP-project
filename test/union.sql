CREATE DATABASE OOP;
USE OOP;
CREATE TABLE oop_info(stu_id INT NOT NULL, PRIMARY KEY(stu_id), stu_name CHAR);
INSERT INTO oop_info(stu_id, stu_name) VALUES (2018011243, "a");
INSERT INTO oop_info(stu_id, stu_name) VALUES (2018011344, "b");
INSERT INTO oop_info(stu_id, stu_name) VALUES (2018011445, "c");
CREATE TABLE oop_info1(stu_id INT NOT NULL, PRIMARY KEY(stu_id), stu_name CHAR);
INSERT INTO oop_info1(stu_id, stu_name) VALUES (2018011243, "a");
INSERT INTO oop_info1(stu_id, stu_name) VALUES (2018011777, "b");
INSERT INTO oop_info1(stu_id, stu_name) VALUES (2018011666, "c");
SELECT stu_id from oop_info union SELECT stu_id from oop_info1;
SELECT stu_id from oop_info union all SELECT stu_id from oop_info1;
DROP DATABASE OOP;
