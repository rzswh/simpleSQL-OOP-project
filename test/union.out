stu_id
2018011243
2018011344
2018011445
2018011666
2018011777
stu_id
2018011243
2018011344
2018011445
2018011243
2018011666
2018011777
